### PR TITLE
8.0 connector ecommerce

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

File: connector_ecommerce/sale.py [hunk 1]
```diff
@@ -291,12 +291,10 @@ def action_view_parent(self, cr, uid, ids, context=None):
         action['res_id'] = parent.id
         return action
 
-    # TODO: remove in odoo 8.0
+    # XXX the 3 next methods seems very specific to magento
```
Comment (guewen) [inline connector_ecommerce/sale.py:294]: Is there a special reason for removing the `TODO: remove in odoo 8.0`?
The methods are really to remove (I will do it in another PR I think)

**Comment by guewen** [comment]
Thanks! I'll have a look tomorrow

File: connector_ecommerce/sale.py (hunk 1 above)
Comment (pedrobaeza) [inline connector_ecommerce/sale.py:294]: Not really, but I have an obsolete copy and I overwrite the code. I haven't done the migration, and don't have a look in depth of the code, so I'm going also to act here as a reviewer/fixer :wink:

Comment (guewen) [inline connector_ecommerce/sale.py:294]: Perfect!

**Comment by guewen** [comment]
Good work has been done, it could have been even better if it was split in several commits.

**Comment by guewen** [comment]
As the migration is based on an obsolete version, it is really hard to see what is changed because it was not there in the migrated code or because it has intentionally be removed for the migration.

I will propose a new PR

**Comment by lepistone** [comment]
I'm not sure it will work in that case, but in https://github.com/OCA/account-financial-tools/pull/145 I did a `rebase -X ours` that seems having worked fine (with `-X`, conflicts are resolved automatically with a preference on one of the two branches).

**Comment by guewen** [comment]
Working on https://github.com/OCA/connector-ecommerce/pull/18

**Comment by pedrobaeza** [comment]
Do we close this then?

**Pull request closed without merging**